src/dns_parser.rs

```diff
@@ -20,6 +22,34 @@ use std::{
     time::SystemTime,
 };
 
+/// InterfaceId is used to represent the interface indentifer
```

Comment:
typo: "indentifer" -> "identifier"

Reply:
Fixed with https://github.com/keepsimple1/mdns-sd/pull/342/commits/35116be8e22fda1dbae5d9a9514f25179c6ce3f0, thanks for reviewing.

Reply:
I also fixed some more minor spelling issues with `typos-cli` in https://github.com/keepsimple1/mdns-sd/pull/342/commits/ba742edaf0105cdb728ca5da9f63b4d2a3e3d71a